docs/tutorials/index.md

```diff
@@ -17,14 +17,15 @@ These tutorials introduce fundamental concepts in deep learning and their realiz
    basic/data
 ```
 
-### Computer Vision
+### MXNet: Training and Inference
```

Comment:
@piiswrong We need to find a place for Linear Regression and I didn't think **Basics** was the right place for it. That is way I suggested renaming this. Later when we add more tutorials, we can revive Computer Vision section.

Do you have concerns?